### ISSUE TITLE
addStylesheet with link to StyleSDK.css isn't UpperCamelCase for StyleSheet

### ISSUE THREAD

**Issue by thorstenBorn-cosmoblonde** [pixx-io/typo3#8]
Plugin: pixxio/pixxio-extension
Version: 2.0.0 

Symptom: The iFrame for Pixxio image selection is too small (wrong height). No lightbox appears.
Cause: The path to StyleSDK.css is written incorrectly. The height definition is therefore missing. 

Solution: Adapt the directory name in the link. /StyleSheet/ instead of /Stylesheet/ 

Classes/Backend/InlineControlContainer.php # 104
```php
$assetsCollector->addStylesheet('pixxio_extension','EXT:pixxio_extension/Resources/Public/StyleSheet/StyleSDK.css');
````

Classes/Controller/FilesControlContainer.php # 230
```php
$assetsCollector->addStylesheet('pixxio_extension','EXT:pixxio_extension/Resources/Public/StyleSheet/StyleSDK.css');
```

**Comment by machwert** [comment]
Thanks @thorstenBorn-cosmoblonde,
Fixed now in dev branch. Will be soon merged to master.

**Comment by rogervanwile** [comment]
Hi @thorstenBorn-cosmoblonde ,

thanks for creating the bug report. We published a new version 2.0.1 with changes related to the issue. Please update your installation.
